hermes-guide-days-website/src/app/footer/footer.component.html

```diff
@@ -0,0 +1,44 @@
+<link
+  rel="stylesheet"
+  href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
i don't see them used, although it would be nice to use them instead of static images, add the line to index.html so they are available globally